portfolio/src/main/webapp/style.css

```diff
@@ -2,8 +2,144 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  min-height: 1000px;
+  background-color: #303030;
+  padding: 50px;
+  color: white;
+  border-width: 2px;
+  border-color: black;
+  border-radius: 5px;
+  border-style: solid;
+  margin-top: 20px;
+  font-family: Arial, Helvetica, sans-serif;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.avatar img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  max-height: 100px;
+  max-width: 100px;
+  margin-bottom: 20px;
+}
+
+.pic img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 300px;
+}
+
+.vid {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+#chessimages img {
+  margin-left: auto;
+  margin-right: auto;
+  height: 300px;
+  width: 300px;
+}
+
+body {
+  background: #303030 url("images/bk.jpg");
+  background-size: 105%;
+  margin: 0px;
+}
+
+.topbar {
+  background-color: #303030e0;
+  width: 100%;
+  min-height: 150px;
+  font-size: 40px;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.topbar .name {
+  right: -25px;
+  position: relative;
+  padding-top: 25px;
 }
+
+.menu, .otherlinks {
+  padding: 10px;
+  width: 50%;
+  font-size: 20px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.menu a {
+  position: relative;
+  right: -20px;
+  float: left;
+  margin: 5px;
+  padding: 10px 15px;
+  background-color: #303030;
+  color: white;
+  text-decoration: none;
+  border-radius: 5px;
+  border-width: 1px;
+  border-color: white;
+  border-style: solid;
+}
+
+.otherlinks a {
+  float: right;
+  margin: 5px;
+  padding: 10px 15px;
+  background-color: #303030;
+  color: white;
+  text-decoration: none;
+  border-radius: 5px;
+  border-width: 1px;
+  border-color: white;
+  border-style: solid;
+}
+
+.menu a:hover, .otherlinks a:hover {
+  background-color: darkorange;
+  color: white;
+}
+
+.linklist {
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 15px;
+  margin-bottom: 15px;
+  width: 30%;
+  text-align: center;
+  font-size: 20px;
+}
+
+.linklist a {
+  padding: 10px;
+  display: block;
+  background-color: #303030;
+  color: white;
+  text-decoration: none;
+  border-radius: 5px;
+  border-width: 1px;
+  border-color: white;
+  border-style: solid;
+}
+
+.linklist a:hover {
+  background-color: darkorange;
+  color: white;
+}
+
+#number-button button {
+  background-color: darkorange;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+  border-width: 0px;
+  border-radius: 2px;
+  font-size: 15px;
+}
+
+a {
+  color: gold;
+}
```

Comment:
Please fix -- "No newline at end of file"

Reply:
Thanks; fixed.